### ISSUE TITLE
Move `tx-type` setting to consensus governed chain definition

### ISSUE THREAD

**Issue by byte-bandit** [VolumeFi/paloma#1512]
At the moment, the `tx-type` of each remote chain is configured by each relayer individually inside `~/.pigeon/config.yaml`.

This can lead to severe misconfigurations, as a chain usually only supports type `0` or `2`, and the entire relayer network should stick to one of these per chain.

Therefore, I suggest moving this setting away from the individual Pigeon configuration, and instead add it to the chain configuration stored directly on Paloma, which is added by governance. That way, when Pigeon queries for chain infos, it can also retrieve the TX Type setting, we don't have to hardcode any information and it becomes owned by governance.

## Acceptance criteria
- [ ] `tx-type` setting is removed from Pigeon's configuration
- [ ] `tx-type` setting is added to Paloma's remote chain configurations, including proposals
- [ ] A new proposal is added to set this setting for existing chains
- [ ] The proposal to support a new chain is updated to include this setting
- [ ] Pigeon queries the setting from Paloma and applies it to it's running configuration

## Rollout plan
1. Update Paloma to support the settings
2. Once deployed, add the `tx-type` to all existing chain configurations using governance proposals
3. Once updated, the Pigeon update can be rolled out

## Estimate
`2-3d` for engineering, not accounting rollout phase

**Comment by taariq** [comment]
I like this, but if we are still not relaying when all validators are updated, this effort won't fix the current issue.

**Comment by byte-bandit** [comment]
> I like this, but if we are still not relaying when all validators are updated, this effort won't fix the current issue.

Depends on whether relayers actually configure their Pigeon correctly or not. This will help with that. But yeah, it's more geared towards ensuring this situation can't arise again in the future.

**Comment by taariq** [comment]
Closing. This is not what is causing our problem. We broke our own relays. Validator have the correct configuration and have upgraded.

**Comment by byte-bandit** [comment]
Reopening this for the backlog. This is about something else, and we should not forget about it.

**Comment by byte-bandit** [comment]
> @byte-bandit okay to close? This was definitely our issue, not validator performance.

As I wrote above, it's just something we should keep in the backlog for now. Given that I've done a proper writeup of what needs doing, I suggest we simply keep it open for now. It would also be a good first issue for Luis.

**Comment by taariq** [comment]
@byte-bandit I agree to put this into governance. If all pigeons vote on it, then it will work.
